src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -92,20 +92,185 @@
+                      <span class="me-2"
+                            *ngIf="title === 'Data Sync'">
+                        <ng-template #syncPopover>
+                          <ul class="text-center">
+                            <li><h5><b>Sync Status:</b></h5></li>
+                            <li *ngFor="let status of zone.fullSyncStatus">
+                              <span *ngIf="!status.includes(zone.name) && (!status.includes(zone.syncstatus) || replicaZonesInfo.length === 2)">
+                                <span *ngIf="status.includes(':')">
+                                    <b>{{ status.split(':')[0] }}</b>:{{ status.split(':')[1]}}
+                                </span>
+                                <span *ngIf="!status.includes(':')">
+                                  <b>{{ status }}</b>
+                                </span>
+                              </span>
+                            </li>
+                          </ul>
+                        </ng-template>
+                        <ul class="me-2">
+                          <li><b>Status:</b></li>
+                          <li><a class="lead text-primary"
+                                 [ngbPopover]="syncPopover"
+                                 placement="top"
+                                 popoverClass="rgw-overview-card-popover"
+                                 i18n>{{ zone.syncstatus }}</a></li>
+                          <li class="mt-4 set-bar-width text-center"
+                              *ngIf="zone.syncstatus === 'preparing for full sync'">
+                            <b>Full sync progress:</b>
+                            <cd-usage-bar *ngIf="zone.fullSync"
+                                          [total]="zone.fullSync[1]"
+                                          [showMultisiteTooltip]="true"
+                                          [used]="zone.fullSync[0]"
+                                          [title]="shards"
+                                          decimals="2">
+                            </cd-usage-bar></li>
+                          <li class="mt-4 set-bar-width text-center"
+                              *ngIf="zone.incrementalSync !== ''">
+                            <b>Sync Progress:</b>
+                            <cd-usage-bar *ngIf="zone.incrementalSync"
+                                          [total]="zone.totalShards"
+                                          [showMultisiteTooltip] ="true"
+                                          [used]="zone.usedShards"
+                                          [title]="shards"
+                                          decimals="2">
+                            </cd-usage-bar></li>
+                        </ul>
+                      </span>
+                    </cd-card>
+                  </div>
+                </cd-card>
+              </div>
+            </cd-card>
+          </div>
+        </div>
+      </div>
+    </cd-card>
+  </div>
```

Comment:
I think we need a generic component for the `card-block` structure which would improve the readability of this section and also maybe useful for any other UIs that we are gonna develop in future

Reply:
what about this?

Reply:
i still don't see if you are planning for this now or later? @aaSharma14